pkg/verify/signature_test.go

```diff
@@ -113,3 +120,50 @@ func TestSignatureVerifierMessageSignature(t *testing.T) {
 	assert.Error(t, err)
 	assert.Nil(t, result)
 }
+
+type nonExpiringVerifier struct {
```

Comment:
This struct and the `trustedPublicKeyMaterial` helper is just copied from `sigstore-go/main.go` which isn't great. Not 100% sure what to do here because we need this to test the relevant functionality, however it probably doesn't belong in the public API.